### PR TITLE
Pm 2021 07 styling issue

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

**Comment by phillimorland** [comment]
for the 10px spacing, I tried changing it but it's everywhere so will leave it for now, I have made an issue to move away from all these various spacing units

File: euth_wagtail/assets/scss/components/_poll.scss [hunk 1]
```diff
@@ -130,14 +130,21 @@ $checkbox-size: 25px;
     align-items: center;
 }
 
+.poll__btn--link {
+    text-decoration: underline;
+    margin-left: 10px;
```
Comment (khamui) [inline euth_wagtail/assets/scss/components/_poll.scss:135]: could be spacer as well?

**Review by khamui — COMMENTED** [review]
two little things, but lgtm!

File: euth_wagtail/assets/scss/components/_poll.scss (hunk 1 above)
Comment (phillimorland) [reply euth_wagtail/assets/scss/components/_poll.scss:135]: see comment

File: euth_wagtail/assets/scss/components/forms/_editpoll.scss [hunk 2]
```diff
@@ -57,6 +59,27 @@
...
     position: relative;
+
+    .errorlist {
+        display: block;
+        position: relative;
+        background-color: $brand-danger;
+        color: $body-bg;
+        font-weight: 600;
+        text-align: right;
+        padding: 4px 8px 4px 28px;
```
[the remark this answers is not in the document]
Comment (phillimorland) [reply euth_wagtail/assets/scss/components/forms/_editpoll.scss:70]: copied directly from our form errors, I have created an issue, see comment

**Comment by khamui** [comment]
> for the 10px spacing, I tried changing it but it's everywhere so will leave it for now, I have made an issue to move away from all these various spacing units
> 
> @khamui this comment :p

ah yeah :D wasn't aware of that. then its fine and we tackle that all at once.

**Pull request merged**